webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/services/reports/ReportingServiceImpl.scala

```diff
@@ -294,20 +295,18 @@ trait CachedFindRuleNodeStatusReports extends ReportingService with CachedReposi
    * Clear cache. Try a reload asynchronously, disregarding
    * the result
    */
-  override def clearCache(): Unit = this.synchronized {
+  override def clearCache(): Unit = {
```

Comment:
why not using the semaphore here ?

Reply:
I think it was useless to synch on that. The clear cache is just the first op (`cache = Map()`). 
The other part is just "reload the cache for the new time". But the part with the complex logic is `checkAndUpdateCache`, and that part is guarded by a semaphore. As far as I can see, the only thing that could happen is a secon call to `checkAndUpdateCache`, which is idempotent.

Reply:
ok, make sense